src/network.js

```diff
@@ -480,15 +480,23 @@ export class BlockstackNetwork {
       reverseHash.reverse()
       excludeSet.push({
         tx_hash: reverseHash.toString('hex'),
-        tx_output_n: utxoUsed.index 
+        tx_output_n: utxoUsed.index
       })
     })
 
     this.excludeUtxoSet = excludeSet
 
     const txHash = tx.getHash().reverse().toString('hex')
     tx.outs.forEach((utxoCreated, txOutputN) => {
-      if (bitcoinjs.script.classifyOutput(utxoCreated.script) === 'nulldata') {
+      const isNullData = function isNullData(script) {
+        try {
+          bitcoinjs.payments.embed({ output: script }, { validate: true })
```

Comment:
the `bitcoin.payments.{embed,p2sh,p2pkh,...}` functions exist now instead of the `bitcoin.script.***.encode` functions.
So here we are creating a nullData output. If that fails it falls through into the `catch {..}` clause.